src/main/frontend/components/NavBar.js

```diff
@@ -8,8 +8,35 @@ import EditVenue from "./EditVenue";
 import VenueShow from "./VenueShow.js";
 
 const NavBar = (props) => {
+
+const [concertVenues, setConcertVenues] = useState([])
+const venueName = props.match.params.type
+
+const fetchConcertVenues = async () => {
+  try {
+    const response = await fetch(`/api/v1/venues/${venueId}`);
+    if (!response.ok) {
+      const errorMessage = `${response.status} (${response.statusText})`;
+      const error = new Error(errorMessage);
+      throw error;
+    }
+  } catch (error) {
+    console.error(`There was an error in fetch: ${error}`);
+  }
+}
+
```

Comment:
I'm still a bit confused as to what the point of 12-27 is.  It looks to set a piece of state and a variable and then make a fetch call, but does nothing with the response